client/components/workouts/LogWorkout.jsx

```diff
@@ -0,0 +1,278 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { auth } from "../../src/firebase";
+import "./LogWorkout.css";
+
+
+export default function LogWorkout() {
+    const [selectedDate, setSelectedDate] = useState(new Date());
+    const [loading, setLoading] = useState(false);
+    const navigate = useNavigate();
+
+    const [workouts, setWorkouts] = useState({
+        cardio: [],
+        strength: [],
+        flexibility: [],
+        sports: []
+    });
+
+    const dailyGoals = {
+        calories: 2800,
+        duration: 60, // will change depending on their onboarding choice
+        exercises: 5,
+    };
+
+    const calculateTotals = () => {
+        let totals = { calories: 0, duration: 0, exercise: 0 };
```

Comment:
Later in the code, `totals.exercises` is updated, but it's initialized as `totals.exercise` earlier on - which may be a typo.

```suggestion
        let totals = { calories: 0, duration: 0, exercises: 0 };
```